MediaBrowser.Common/Json/JsonDefaults.cs

```diff
@@ -56,6 +56,7 @@ public static JsonSerializerOptions GetCamelCaseOptions()
         {
             var options = GetOptions();
             options.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
+            options.PropertyNameCaseInsensitive = true;
```

Comment:
This should be placed on the generic `GetOptions`, and also mapped in `AddJellyfinApi`

Reply:
done